content/sensu-go/6.2/operations/monitoring-as-code/_index.md

```diff
@@ -0,0 +1,151 @@
+---
+title: "Monitoring as Code with Sensu"
+linkTitle: "Monitoring as Code"
+description: "Read this page to learn about following a monitoring-as-code approach with Sensu."
+product: "Sensu Go"
+version: "6.2"
+weight: 5
+layout: "single"
+toc: true
+menu:
+  sensu-go-6.2:
+    parent: operations
+    identifier: monitoring-as-code
+---
+
+Sensu supports end-to-end monitoring as code so you can manage your observability and monitoring workflows the same way you build, test, and deploy your applications and infrastructure.
+Sensu allows you to define your entire monitoring workflow as declarative YAML or JSON code in configuration files, from collecting observability events and metrics and diagnosing issues through sending alerts and automatically remediating problems.
+
+In the monitoring as code approach, when a new endpoint starts up, like a cloud compute instance or Kubernetes Pod, Sensu automatically registers itself with the platform and starts collecting monitoring and observability data according to the code in your configuration files.
+If you manage your monitoring and observability workflow in the same way as the systems you're monitoring &mdash; as code, using a centralized continuous integration/continuous delivery (CI/CD) pipeline &mdash; you can align your monitoring with your product and services and improve visibility, reliability, and repeatability.
+
+- Your entire team can share, edit, review, and version Sensu configuration files just like they would with other "as-code" solutions.
+- Export your Sensu configuration from one environment and initialize another environment with the same configuration.
+- Use Sensu configuration files to remove, restore, back up, and recover Sensu instances.
+
+## Create a monitoring as code repository
+
+You can use any source control repository for maintaining your monitoring workflow configuration files.
+Your monitoring as code repository should include configuration files with the resource definitions you use in your monitoring workflow.
+
+Use `sensuctl dump` to [export some or all of your existing Sensu resource definitions][7] to standard out (STDOUT) or to a file.
```

Comment:
We talked about using "export" and "initialize" instead of "backup" and "recover" for this scenario.

We could:
- Reword and rename https://docs.sensu.io/sensu-go/latest/sensuctl/back-up-recover/ to use "export" and "restore" and "initialize"
- Recreate a doc similar to https://docs.sensu.io/sensu-go/latest/sensuctl/back-up-recover/ but focused on the monitoring as code use case (exporting and reinitializing)

I think the second option is better. I'm not sure if it needs to be a separate document inside a "Monitoring as Code" sub-category or if it would be brief enough to include here on this page.

Reply:
I've been thinking about this some more. I think the best option is including an "export and reproduce" section here on the MaC page.